src/components/Article/index.tsx

```diff
@@ -4,7 +4,7 @@ import { ArticleFooter } from './ArticleFooter';
 
 const Article: FC<{ hasTopBar?: boolean }> = ({ children, hasTopBar = false }) => (
   <article
-    className={cn('md:mt-12 w-full px-24 md:pl-40 md:pr-48 xl:pr-64 col-start-2 xxl:col-start-1', {
+    className={cn('md:mt-12 w-full px-24 md:pl-40 md:pr-48 xl:pr-64 col-start-2 xxl:col-start-1 overflow-x-hidden', {
```

Comment:
This seems like it will have a lot more consequences than what is intended here?